config.example.wikimedia.yaml

```diff
@@ -450,6 +443,170 @@ templates:
                 request:
                   uri: https://wikimedia.org/api/rest_v1/metrics/pageviews/top/{+rest}
 
+      /{module:graphoid}/v1/store/{format}:
```

Comment:
We really need a way to pull in spec fragments like this from a separate file, passing in a config object.